### ISSUE TITLE
fweb docs wrong for  clock.schedule_interval

### ISSUE THREAD

**Issue by profK** [lordmauve/pgzero#302]
Example on the web docs uses fractional seconds but the actual call parameter is in milliseconds

**Comment by lordmauve** [comment]
I don't think it does use milliseconds? Code and docs agree that it uses seconds; `dt` is converted to seconds at the earliest opportunity.
